env0/resource_environment.go

```diff
@@ -248,6 +253,12 @@ func resourceEnvironmentUpdate(ctx context.Context, d *schema.ResourceData, meta
 }
 
 func resourceEnvironmentDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	canDestroy := d.Get("force_destroy")
+
+	if canDestroy != true {
+		return diag.Errorf(`must enable "force_destroy" safeguard in order to destroy`)
+	}
```

Comment:
do you sure about it? this means if the user didn't set force_destroy: true he can't destroy this environment

Reply:
yea, thats what @avnerenv0 and @yaronya wanted
They said thats how it is in s3 tf provider

Reply:
mozar - The user should know when he does destroy the data will be deleted

Reply:
@avnerenv0 do we sure about it for destory?

Reply:
Yeah man we had like 50 discussions about this already.

This concept comes from the `aws_s3_bucket` resource and is pretty standard and accepted in TF.
Also exists for `google_storage_bucket`, for example.

There is also `prevent_destroy` concept but we'd like to go with the 'safer' option.